infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryController.kt

```diff
@@ -202,7 +200,7 @@ class GeometryController @Autowired constructor(
         return toFileDownloadResponse(filename.withSuffix(CSV), content.toByteArray(Charsets.UTF_8))
     }
 
-    @PreAuthorize(AUTH_UI_READ)
+    @PreAuthorize(AUTH_VIEW_INFRAMODEL)
```

Comment:
Näistä kyselinkin Slackissa. Näin nämä oli speksattu, mutta onkohan varmasti haluttua, että myös paikannuspohjaperustaisilla tietotuotteilla tutkitaan `AUTH_VIEW_INFRAMODEL`-oikkaa? Omasta mielestäni esim. `AUTH_VIEW_LAYOUT` olisi jotenkin luontevampi.